bazel/envoy_build_system.bzl

```diff
@@ -168,7 +168,7 @@ def envoy_cc_test(name,
     native.cc_test(
         name = name,
         copts = envoy_copts(repository),
-        linkopts = ["-pthread"],
+        linkopts = ["-pthread", "-latomic"],
```

Comment:
Maybe just a TODO here to reduce the globalization of pthread and atomic someday. It's probably ine to link atomic with everything for now, but I worry about creep in libs.